client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-key-based-dialog.js

```diff
@@ -0,0 +1,240 @@
+
+		// If we had no keys at mount
+		if ( ! this.state.hadKeysAtStart ) {
+			return (
+				<div className="stripe__method-edit-header">
+					{ translate( 'Take credit card payments with Stripe' ) }
+				</div>
+			);
+		}
+
+		return (
+			<div className="stripe__method-edit-header">
+				{ translate( 'Manage Stripe' ) }
+			</div>
+		);
+	}
+
+	onSelectLive = () => {
+		this.props.onEditField( { target: { name: 'testmode', value: 'no' } } );
+	}
+
+	onSelectTest = () => {
+		this.props.onEditField( { target: { name: 'testmode', value: 'yes' } } );
+	}
+
+	renderModePromptAndKeyFields = () => {
+		const { method, translate } = this.props;
+
+		let keys = [];
+
+		if ( method.settings.testmode.value === 'no' ) {
+			keys = [
+				{
+					id: 'secret_key',
+					label: translate( 'Live Secret Key' ),
+					placeholder: translate( 'Enter your secret key from your Stripe.com account' ),
+					value: method.settings.secret_key.value,
+				},
+				{
+					id: 'publishable_key',
+					label: translate( 'Live Publishable Key' ),
+					placeholder: translate( 'Enter your publishable key from your Stripe.com account' ),
+					value: method.settings.publishable_key.value,
+				}
+			];
+		} else {
+			keys = [
+				{
+					id: 'test_secret_key',
+					label: translate( 'Test Secret Key' ),
+					placeholder: translate( 'Enter your secret key from your Stripe.com account' ),
+					value: method.settings.test_secret_key.value,
+				},
+				{
+					id: 'test_publishable_key',
+					label: translate( 'Test Publishable Key' ),
+					placeholder: translate( 'Enter your publishable key from your Stripe.com account' ),
+					value: method.settings.test_publishable_key.value,
+				}
+			];
+		}
+
+		return (
+			<div>
+				<FormFieldset className="stripe__method-edit-field-container">
+					<TestLiveToggle
+						isTestMode={ 'yes' === method.settings.testmode.value }
+						onSelectLive={ this.onSelectLive }
+						onSelectTest={ this.onSelectTest }
+					/>
+				</FormFieldset>
+				<APIKeysView
+					highlightEmptyRequiredFields={ this.state.highlightEmptyRequiredFields }
+					keys={ keys }
+					onEdit={ this.props.onEditField }
+				/>
+			</div>
+		);
+	}
+
+	onSelectAuthOnly = () => {
+		this.props.onEditField( { target: { name: 'capture', value: 'no' } } );
+	}
+
+	onSelectCapture = () => {
+		this.props.onEditField( { target: { name: 'capture', value: 'yes' } } );
+	}
+
+	getStatementDescriptorPlaceholder = () => {
+		const { site, translate } = this.props;
+		const domain = site.domain.substr( 0, 22 ).trim().toUpperCase();
+		return translate( 'e.g. %(domain)s', { args: { domain } } );
+	}
+
+	renderMoreSettings = () => {
+		const { method, translate } = this.props;
+
+		return (
+			<div>
+				<AuthCaptureToggle
+					isAuthOnlyMode={ 'no' === method.settings.capture.value }
+					onSelectAuthOnly={ this.onSelectAuthOnly }
+					onSelectCapture={ this.onSelectCapture }
+				/>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Descriptor' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **44** times:
`translate( 'Descripton' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).